### PR TITLE
Fix emails validation with unknown TLD

### PR DESCRIPTION
**Pull request by barbushin**
refs #8391

### CODE REVIEW

File: config/global.php [hunk 1]
```diff
@@ -66,4 +66,10 @@
 
     'Piwik\EventDispatcher' => DI\object()->constructorParameter('observers', DI\get('observers.global')),
 
+    'Zend_Validate_EmailAddress' => function () {
+        return new \Zend_Validate_EmailAddress(array(
+            'hostname' => new \Zend_Validate_Hostname(array(
+                'tld' => false,
```
Comment (mattab) [inline config/global.php]: are there other settings we could disable from email validation, to try and accept as many 'valid' emails as possible?

Comment (barbushin) [inline config/global.php]: I tested all available settings, and I think `tld=false` is the only one we need to use. There was a problem with out-dated list of TLD zones that Zend used to validate email.

Comment (mattab) [inline config/global.php]: cheers, sounds good  :+1:

**Comment by mnapoli** [comment]
:+1: failed UI tests are because of outdated UI screenshots (probably the branch was created before those screenshots were updated)

**Pull request merged**